indico/modules/events/editing/client/js/components/FileManager/util.js

```diff
@@ -62,7 +62,7 @@ export function uploadFiles(action, fileTypeId, acceptedFiles, uploadURL, dispat
         action(fileTypeId, fileId, tmpFileId, {
           filename: uploadedFile.filename,
           uuid: uploadedFile.uuid,
-          downloadURL: null,
+          downloadURL: uploadedFile.downloadURL,
```

Comment:
Is this intended? We only have a download URL for files that are already attached to a revision (we could probably create a blob url from the locally-available file data until then).

Reply:
I added it so that we can at least mock the response from the server and it will work. If there isn't a `downloadURL` we'll get `undefined`, which I guess is OK here.

Reply:
You're right. We will only get `downloadURL` for claimed files, which will never come from this handler.